### ISSUE TITLE
can't install

### ISSUE THREAD

**Issue by ms88privat** [xiongchuan86/react-native-pxplayer#1]
playerxx.h not found.

Thanks!

**Comment by yonigozman** [comment]
did you link it to your project? but even if you do, there are missing libs because they were not committed to git as a part of the library

**Comment by ms88privat** [comment]
@yonigozman yes I did. What can I do about the missing libs? I also can not install react-native-vlc-player. Anyways, what is the difference between the two libs?

can any of these libs play udp multicast streams?

Thanks
